testsuite/tests/kuadrant/reconciliation/conftest.py

```diff
@@ -0,0 +1,24 @@
+"""Conftest for reconciliation tests"""
+import backoff
+import pytest
+
+
+@pytest.fixture(scope="module", autouse=True)
+def commit(request, authorization, kuadrant):
+    """Commits all important stuff before tests"""
+    if not kuadrant:
+        pytest.skip("Reconciliation test can only run on Kuadrant for now")
+    request.addfinalizer(authorization.delete)
+    authorization.commit()
+
+
+@pytest.fixture(scope="module")
+def resilient_request(client):
```

Comment:
`HttpxBackoffClient` is doing the same thing no?

Reply:
Not exactly, `HttpxBackoffClient` is currently only for backoff for 503 (we backoff no and has much higher timeout period, technically the `HttpxBackoffClient` could be updated to support even these use case if you would like so

Reply:
I added `add_retry_code` in the past, where you can adjust retry codes.

Reply:
You can, but I reuse the client and I would have to remove it afterwards, which seems suboptimal to say the least. I also tried to add to Client constructor but that would mean I have to create new client for each request (basically copying the client fixtures), which is also not great.